### PR TITLE
fix(GalleryItem): fix image heigh on gallery Item

### PR DESCRIPTION
**Pull request by roiLeo**
Only for testing purposes

### CODE REVIEW

File: src/components/rmrk/Gallery/GalleryItem.vue [hunk 1]
```diff
@@ -334,7 +334,10 @@ hr.comment-divider {
     margin: 30px auto;
     width: 100%;
 
-    .image {
+    .nft-image img {
+      max-height: 530px;
```
Comment (vikiival) [inline src/components/rmrk/Gallery/GalleryItem.vue:338]: I think this will be broken on mobile

Comment (roiLeo) [reply src/components/rmrk/Gallery/GalleryItem.vue:338]: ### Screenshot
#### Mobile View large image
Screenshot 2021-09-27 at 17-17-55 Decorator Tiles #2 - Dune special
#### Mobile View long image
Screenshot 2021-09-27 at 17-20-29 Kusamagen #160

**Comment by vikiival** [comment]
It would be nice to have a fixed max height so the layout would look the same.
As you can see on the volcano user is able to see descriptions and so on. On the other hand on kusamagen you can see name, collection, and creator (which is correct).

Screenshot 2021-09-30 at 09 43 09

Screenshot 2021-09-30 at 09 43 43

**Comment by yangwao** [comment]
> It would be nice to have a fixed max height so the layout would look the same. As you can see on the volcano user is able to see descriptions and so on. On the other hand on kusamagen you can see name, collection, and creator (which is correct).
> 
> Screenshot 2021-09-30 at 09 43 09
> 
> Screenshot 2021-09-30 at 09 43 43

Make sense to have some fixed height where art is placed. That was original thoughts like foundation is doing it tho. I guess it could be as option or preference for user what they look like to have there :)

**Comment by yangwao** [comment]
Should we take this in or it's already obsolete?

**Comment by roiLeo** [comment]
closed in favor of #814, #813 we will maybe need a fixed height later for GalleryItem

**Pull request closed without merging**